### PR TITLE
Bump NWjs to 0.54.1

### PR DESCRIPTION
**Pull request by haslinghuis**
Bumps NWjs from 0.54.0 to 0.54.1 fixing an issue with not being able to select a custom image using Font Manager.

Also tried 0.55.0 but @kevinsummer pointed out a problem with registering the Notification Center App ID as `io.nwjs.nwjs.framework.alertnotificationservice` instead of `com.nw-builder.betaflight-configurator`.

See https://github.com/betaflight/betaflight-configurator/issues/2274

### CODE REVIEW

**Comment by McGiverGim** [comment]
We try to use same version for Configurator and Blackbox. Is there a similar PR in the other repo?

**Comment by haslinghuis** [comment]
Working on it now

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
Because Mikellers 0.54 PR not yet merged :(
https://github.com/betaflight/blackbox-log-viewer/pull/518

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Comment by Asizon** [comment]
@haslinghuis why you are not using 0.55 here and in bb part? seems that ffmpg 0.55 library version is avaiable.

**Comment by haslinghuis** [comment]
Using the ffmpeg prebuilt library 0.55.0 now.
NWjs 0.55.0 was not working as 0.54.1 does.
Also tested with Node 16 on Linux without problem. Windows and OSX is not working with Node 14 / 16.

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by mikeller — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
